packages/block-library/src/media-text/editor.scss

```diff
@@ -1,34 +1,46 @@
-.wp-block-media-text,
-.wp-block-media-text.aligncenter {
-	grid-template-areas:
-		"media-text-media media-text-content"
-		"resizer resizer";
-}
-
-.wp-block-media-text.has-media-on-the-right,
-.wp-block-media-text.has-media-on-the-right.aligncenter {
-	grid-template-areas:
-		"media-text-content media-text-media"
-		"resizer resizer";
-}
-
 .wp-block-media-text .__resizable_base__ {
-	grid-area: resizer;
+	-ms-grid-column: 1;
+	-ms-grid-column-span: 2;
+	-ms-grid-row: 2;
```

Comment:
Instead of adding this mse specific styles here, can we just enable `grid: true` in autoprefixer config?

Reply:
Hi @youknowriad, I agree using the autoprefixer is an awesome idea and makes the code simpler.
I did some tests, and the template areas we had were not working even with the autoprefixer, setting columns spans using grid-column: 1 / -1; also did not work but using grid-column: 1 / span 2; works.
It looks like with this changes to the CSS and using the autoprefixer things are working fine in more modern browsers and in IE11, without having IE prefixes around the code.
Thank you for the suggestion.

Reply:
I don't see the change to the autoprefixer config in the changes? Is it already enabled?

Reply:
Sorry, I missed a stash on the change, the problem was corrected.